plugin/src/main/scala/scoverage/CoverageFilter.scala

```diff
@@ -28,8 +28,13 @@ object AllCoverageFilter extends CoverageFilter {
 class RegexCoverageFilter(
     excludedPackages: Seq[String],
     excludedFiles: Seq[String],
-    excludedSymbols: Seq[String]
+    excludedSymbols: Seq[String],
+    reporter: scala.tools.nsc.reporters.Reporter
```

Comment:
I don't think we need to inline the fully qualified import here. Can we just import it up top and use `Reporter` here instead?

Reply:
Yes. Done. Fixed.